main.go

```diff
@@ -350,8 +350,15 @@ func _main() error {
 	stdin := false
 	pretty := false
 	schema := false
-	cacheSettings := runner.DefaultCacheSettings
-	for _, arg := range os.Args[1:] {
+	sqlFile := ""
+
+	args := os.Args[1:]
+	for i := 0; i < len(args); i++ {
+		arg := args[i]
+
+
+	  cacheSettings := runner.DefaultCacheSettings
```

Comment:
Your tests are failing cuz this shouldn't be inside the for loop. Should still be back at line 353.

Reply:
Yeah. Sorry im not used to the github editor

Reply:
No problem!

Reply:
How do i fix it?

Reply:
How were you editing before? Just pull locally to bring your laptop back up to date then edit it with a new commit.

I'm not too concerned about commit hygiene because all commits will be squashed when merged.